python/lsst/pipe/tasks/measurePsf.py

```diff
@@ -238,7 +237,7 @@ def __init__(self, schema=None, **kwargs):
         self.makeSubtask("psfDeterminer", schema=schema)
 
     @pipeBase.timeMethod
-    def run(self, exposure, sources, expId=0, matches=None):
+    def run(self, exposure, sources, expId=1, matches=None):
         """!Measure the PSF
```

Comment:
I did not actually think that there was a code path that would produce expId=0, except maliciously.  But I agree that it could happen.  I do think that setting the reserveSeed to 0 would not happen unless the person who configures it did it deliberately.